src/index.ts

```diff
@@ -30,8 +32,22 @@ async function run(): Promise<void> {
     await KubeConfig.writeOutKubeConfig();
 }
 
-run()
-    .then(() => {
-        ghCore.info("Success.");
-    })
-    .catch(ghCore.setFailed);
+async function logout(): Promise<void> {
+    await Auth.logout();
+    await KubeConfig.deleteKubeConfig();
+}
+
+if (!IsPost) {
+    run()
+        .then(() => {
+            ghCore.info("Success.");
+        })
+        .catch(ghCore.setFailed);
+}
+else {
+    const performLogout = ghCore.getBooleanInput("logout");
```

Comment:
Please run `npx action-io-generator --actionYml ./action.yml --outFile ./src/generated/inputs-outputs.ts` so that inputs are updated in `./src/generated/inputs-outputs.ts` and then can be used here as:

```suggestion
    const performLogout = ghCore.getBooleanInput(Inputs.LOGOUT);
```

Reply:
Updates inputs. A bit hard to add a good method of testing a post action, but I set the environment flag manually. That should trigger the post action when calling the action.